ControlService/test/CMakeLists.txt

```diff
@@ -30,4 +30,6 @@ target_link_libraries(${PLUGIN_NAME}
     ${NAMESPACE}Protocols::${NAMESPACE}Protocols
     )
 
+target_compile_options(${PLUGIN_NAME} PRIVATE -Wall -Werror -Wno-error=deprecated-declarations)
```

Comment:
please remove flags from the project files

Reply:
These are plugin specific flags, the global enable & disable will be handled in rdkservices_git.bb.

Reply:
Removed flags from all project files